src/AccessibilityInsights/MainWindow.xaml

```diff
@@ -179,13 +161,30 @@
                                 <behaviors:KeyboardToolTipButtonBehavior/>
                             </i:Interaction.Behaviors>
                             <Grid Width="40" Height="40" x:Name="addAccountGrid">
-                                <Image x:Name="imgAvatar" Visibility="Collapsed" Source="{Binding Path=vmAvatar.ByteData, TargetNullValue={x:Null}}"/>
                                 <Grid x:Name="newAccountGrid" Visibility="Visible">
                                     <AccessText Text="_i" Opacity="0"/>
-                                    <fabric:FabricIconControl GlyphName="Contact" FontSize="24" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=LightIconBrush}"  HorizontalAlignment="Center"/>
+                                    <fabric:FabricIconControl GlyphName="{Binding Path=vmReporterLogo.FabricIconLogoName, TargetNullValue=PlugDisconnected}" FontSize="24" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=LightIconBrush}"  HorizontalAlignment="Center"/>
                                 </Grid>
```

Comment:
Does it mean that we don't show a user profile picture?

Reply:
Correct, it will only be visible in the extension's configurion in the connection settings tab.